go.mod

```
@@ -165,3 +165,5 @@ replace (
 	// use grpc version that's compatible with cosmos-flavored protocol buffers
 	google.golang.org/grpc => google.golang.org/grpc v1.33.2
 )
+
+replace github.com/tendermint/tendermint => github.com/skip-mev/mev-tendermint v0.34.24-0.20221216212242-89790ca280e5
```

```suggestion
replace github.com/tendermint/tendermint => github.com/skip-mev/mev-tendermint v0.34.24-mev.13
```
Should we use a tag instead of a commit hash ?